### PR TITLE
Make rubocop work again

### PR DESCRIPTION
**Pull request by dlackty**
For some reason, Rubocop misconfigured file list with only `Rakefile` & `Gemfile`, so I changed that to fix Rubocop linting and fixed the issues.

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 AllCops:
-  TargetRubyVersion: 2.3
+  TargetRubyVersion: 2.5
```
Comment (grosser) [inline .rubocop.yml]: needs to be 2.3 since that is the minimum supported version

**Comment by grosser** [comment]
nice!

we need to also run rubocop on ci :D

**Comment by grosser** [comment]
... it's already in .travis.yml but it's not getting run :(

**Comment by grosser** [comment]
trying to fix this here https://github.com/ambethia/recaptcha/compare/grosser/rubo?expand=1

**Comment by grosser** [comment]
kk, fixed on master

**Comment by dlackty** [comment]
@grosser Updated & rebased on master!

**Pull request merged**